pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -135,6 +135,13 @@ in
     hardeningDisable = [ "format" ];
   };
 
+  rdiscount = attrs: {
+    # Disable randomized mangling to make outputs reproducible
+    dontBuild = false;
+    buildInputs = [ discount ];
+    patches = [ ./rdiscount-use-nixpkgs-libmarkdown.patch ];
```

Comment:
Is this patch coming from Debian? If yes, adding a link to the source would be nice.

Reply:
The patch is similar but not the same: https://sources.debian.org/src/ruby-rdiscount/2.1.8-1/debian/patches/01_use-system-libmarkdown.patch/

Reply:
It's always good to add links, even if it's just "inspired by <URL>"

Reply:
Going to force-push to add a comment referencing the Debian patch in a second.